src/compas_rhino/scene/meshobject.py

```diff
@@ -37,10 +37,25 @@ class MeshObject(RhinoSceneObject, BaseMeshObject):
     **kwargs : dict, optional
         Additional keyword arguments.
 
+    Attributes
+    ----------
+    disjoint : bool, optional
+        Draw the faces of the mesh disjointed.
+        Default is ``False``.
+    group : str, optional
+        The name of the group to add the mesh components. The group will be created if not already present.
+        Default is ``None``.
+
     """
 
     def __init__(self, mesh, **kwargs):
         super(MeshObject, self).__init__(mesh=mesh, **kwargs)
+        self.disjoint = kwargs.get("disjoint", False)
+        self.group = kwargs.get("group", None)
```

Comment:
Make these two formal attributes here. (another question is that should they go to core?)